api/api/controllers/search_controller.py

```diff
@@ -495,43 +496,42 @@ def search(
     return results, page_count, result_count, search_context.asdict()
 
 
-def related_media(uuid, index, filter_dead):
-    """Given a UUID, find related search results."""
+def related_media(uuid: str, index: str, filter_dead: bool) -> list[Hit]:
+    """Given a UUID, find related search results based on title and tags."""
 
-    search_client = Search(index=index)
+    # Search the default index for the item itself as it might be sensitive.
+    item_search = Search(index=index)
+    item_hit = item_search.query("match", identifier=uuid).execute().hits[0]
```

Comment:
This should use `identifier__keyword` and `term` to avoid stemming on the `text` field that `identifier` is currently. This issue came up during the development of the `SearchContext::build` method where I noticed I wasn't getting the exact list of identifiers back :scream: